bubblewrap.c

```diff
@@ -1458,6 +1485,9 @@ main (int argc,
   /* We want sigchild in the child */
   unblock_sigchild ();
 
+  if (label_exec(opt_exec_label) == -1)
```

Comment:
With the current way we are handling this, the pid1 would not be allowed to be used by the container processes.  pid1 would be running with a unconfined_t process while the confined processes is running with svirt_lxc_net_t.  This means a signal from subprocesses to the parent would be blocked. Doing things like ls /proc/1/* will probably be blocked ...